cmd/kube-proxy/app/server.go

```diff
@@ -595,6 +597,7 @@ func (s *ProxyServer) Run() error {
 		s.Broadcaster.StartRecordingToSink(&v1core.EventSinkImpl{Interface: s.EventClient.Events("")})
 	}
 
+	errCh := make(chan error)
```

Comment:
comments?  This seems out of place here, but if we handle both healthz and metrics, it makes more sense.